### DISCUSSION TITLE
User properties and Firebase Remote Config

### DISCUSSION THREAD

**Discussion by alvindizon** [firebase/firebase-android-sdk#4872] [General]
So I've tried to use a custom user property (`user_code`) as a conditional for my feature toggle in Firebase Remote Config. My use case is, I set the custom user property and fetch and activate (via `fetchAndActivate`) the remote configs on successful log in, and reset the user property by passing null to `setUserProperty` on log out. I also set it on app start up if user is already logged in, alongside with fetching and activate remote configs. 

I intend my feature toggle to be true when `user_code` is equal to a certain value(`abcd`), false if it is not. I can confirm that this is indeed the case, when I log in using the account with `user_code` equal to `abcd`. If I try to log out and log in using a different account, I get the correct feature toggle value (false). But when I try to log in again using the account with `abcd` as the `user_code` , the previous remote config values are still being used, i.e., the feature toggle is still false, when it is supposed to be true. It's only when I restart the app that the correct values are fetched. Has anyone experienced the same?

**Comment by rlazo** [comment]
Hi @alvindizon, if you need to get notified of changes to the remote config values immediately, you can use the https://firebase.google.com/docs/remote-config/real-time?platform=android
